Add licenses documentation

zuazo opened the pull request:
Based on what was discussed in #25.

You can see the preview [here](https://github.com/zuazo-forks/meta/blob/licenses/licenses.md).
- Use [Apache License](http://www.apache.org/licenses/LICENSE-2.0) for software by default.
- Use [Creative Commons Attribution 4.0 International](http://creativecommons.org/licenses/by/4.0/) for documentation by default.
- You are free to use other OSS/CC licenses.

I did not put anything about what we will do if we adopt projects with other licenses. I suppose we can discuss that later.

**Important clarification:** I am not a lawyer.

zuazo committed:
Add licenses documentation

dblessing commented on licenses.md:
```diff
@@ -0,0 +1,305 @@
+
+You can see the different licenses available here: http://choosealicense.com/
+
+How to add the license to a project:
```
For clarity can you change to "How to add the Apache license to a project:"

dblessing commented on licenses.md:
```diff
@@ -0,0 +1,305 @@
+# License and Author
+
+**Original Author:** John Doe
+**Current Author:** [Chef Brigade Team](https://github.com/chef-brigade)
```
I wonder if we should list the current maintainers here as opposed to 'Chef Brigade Team'. Thoughts?

martinb3 replied:
Honestly, I feel like the contributors link on Github is a better way to do this than actually list individuals, e.g. https://github.com/chef-brigade/unbound-cookbook/graphs/contributors

dblessing replied:
That would be a good addition. However, we should explicitly list current maintainers, too. Otherwise it won't be clear who is responsible for issues/PRs

josephholsten commented:
+1 to contributors link and explicitly listing maintainer.

Seems like `LICENSE` and `metadata.rb` should point to chef-brigade, but the `README` should say the specific maintainer. The reasoning being: if someone finds this code two years from now and has a legal issue or bug, they should just come to the brigade and we can point them in the right direction. But if they're already here, they should be able to find the point-of-contact easily in the `README`.

zuazo committed:
license.md improvements based on the feedback:

- Specify license name in the "how to"
- Add author and contributor list to the README

zuazo commented:
OK. Thank you all for your review and your comments.

I think the final README template will go in #12 and in the [cookbook-skeleton](https://github.com/chef-brigade/cookbook-skeleton). This is an example of how you can add the license at the end. Of course, I made all the changes you have proposed because I agree with them. But we will have the chance to discuss it again in other issues :wink:

Please tell me if there is anything else to change or is fine now.

josephholsten commented:
:+1: :shipit: LGTM.

I'll probably extract the huge blocks of legal text into separate files eventually.

zuazo commented:
@dblessing, @martinb3 I'm planning to merge this tomorrow morning if it's ok now.

dblessing commented:
:+1:

josephholsten commented:
yay!
